web/docs/deployment/deployment-methods/self-hosted.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Self-Hosted
+---
+
+If you have your own server or you rent out a server, you can self-host your Wasp apps. This deployment option has the benefit of giving you full control over your apps and its data. Also, it can be more cost-effective than using a cloud provider since you can deploy multiple apps on a single server.
+
+## What you'll need
+
+To successfully self-host your Wasp app, you need to have the following:
+- A server with a public IP address
+  - There are many cloud providers you can use to rent a server. Some popular ones are [AWS](https://aws.amazon.com/ec2/), [DigitalOcean](https://www.digitalocean.com/), [OVH](https://www.ovhcloud.com/en/vps/), and [Hetzner](https://www.hetzner.com/cloud/).
+- A domain name (we'll need this to have HTTPS)
+
+## Self-hosting steps
+
+To self-host your Wasp app, you need to follow these general steps:
+1. Get your app's code on the server. How you do this depends on your setup, we'll explore a few methods in the next section.
+2. Set up [Docker](https://docs.docker.com/engine/install/) on the server. This is needed to run the server and the database.
+3. Set up a reverse proxy to be able to use a domain name with your app.
+4. Set up the server env variables.
+
+![Self-hosting steps](/img/deploying/self-hosting.png)
+
+## Deployment methods
+
+We'll explore a few methods you can use to self-host your Wasp app. One is to use a simple setup where you manually set up the server. The other two methods will use self-hosted PaaS platforms that make it easy to deploy apps.
+
+### Simple setup
+
+In this setup, you install Docker on your server, set up the server env variable, set up a reverse proxy, and run your app. This is a very manual process, but you'll learn how everything works.
+
+<details>
+<summary>Overview of the steps</summary>
+
+1. Install Docker, Node.js and Wasp on your server.
+2. Set up a reverse proxy.
+    - We are using [Caddy](https://caddyserver.com/) in our example, but you can use any reverse proxy you like (e.g., Nginx, Apache).
```

Comment:
Named it "full guide" now since we use that name below.

Reply:
Ok, so in the gist below, got it. I would in this case use `below` becuase otherwise it can be a bit hard to say where it is. I know using `below` is not great, but here it makes sense. Although maybe we can just remove this high level overview as I suggseted above, it seems a bit redundant.